packages/grid/_modules_/grid/hooks/features/virtualization/useGridNoVirtualization.ts

```diff
@@ -69,6 +77,25 @@ export const useGridNoVirtualization = (
     windowRef,
   ]);
 
+  const disableVirtualization = React.useCallback((): void => {
+    setGridState((state) => ({
+      ...state,
+      containerSizes: {
+        ...state.containerSizes!,
+        renderingZone: {
+          ...state.containerSizes!.renderingZone,
+          height: visibleRowCount * props.rowHeight!,
```

Comment:
Same thing as before. Using `props.rowHeight` won't consider different densities. The selector has to be used.